circle.yml

```diff
@@ -25,16 +25,20 @@ dependencies:
     - DEBIAN_FRONTEND=noninteractive sudo apt-get autoremove -y --purge
         libblas-dev
         libbz2-dev
+        libexpat1-dev
```

Comment:
Hmm. Currently we intentionally install python2.7-dev. A reasonable option would be to use the brewed Python for libraries and headers.

[10:22]  
So options are…

1. Install python2.7 and python2.7-dev
2. Install python2.7 but not python2.7-dev
3. Install neither python2.7 nor python2.7-dev

I don't know whether the the :python requirement currently checks that the Python header/libraries are installed. That'd be needed for option 2.

Currently we use option 1.